pkg/controller/kabaneroplatform/tekton.go

```diff
@@ -2,45 +2,32 @@ package kabaneroplatform
 
 import (
 	"context"
+	mf "github.com/jcrossley3/manifestival"
 	kabanerov1alpha1 "github.com/kabanero-io/kabanero-operator/pkg/apis/kabanero/v1alpha1"
-	"github.com/kabanero-io/kabanero-operator/pkg/assets/config"
-	"github.com/kabanero-io/kabanero-operator/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func reconcile_tekton(ctx context.Context, k *kabanerov1alpha1.Kabanero, c client.Client) error {
-	f, _ := config.Open("reconciler/tekton.yaml")
-	defer f.Close()
+	filename := "config/reconciler/tekton.yaml"
+	m, err := mf.NewManifest(filename, true, c)
+	if err != nil {
+		return err
+	}
 
-	if !k.Spec.Tekton.Disabled {
-		log.Info("Tekton is currently enabled")
+	transforms := []mf.Transformer{
+		mf.InjectOwner(k),
+		mf.InjectNamespace(k.GetNamespace()),
+	}
 
-		options := &client.ApplyOptions{
-			OwningController: k,
-			Namespace:        k.GetNamespace(),
-		}
+	m.Transform(transforms...)
```

Comment:
It seems that if resource transformation did not take place, we should not be applying it. It is unclear to me whether or not an m.applyAll failure due to a transformation problem will be obfuscated; therefore, harder to debug.